client/components/content-containers/tcc-assessment-block/edit/index.vue

```diff
@@ -15,60 +15,76 @@
         :key="it._cid"
         @selected="toggleSelect(it)"
         @save="saveAssessment"
-        @delete="requestRemoveConfirmation(it)"
+        @delete="$emit('deleteElement', it)"
         :assessment="it"
         :expanded="isSelected(it)" />
     </ul>
     <add-element
       @add="addAssessment"
       :include="['ASSESSMENT']"
-      :activity="activity"
+      :activity="container"
+      :position="nextPosition"
       :layout="false"
       large
       label="Add assessment" />
   </div>
 </template>
 
 <script>
-import { mapActions, mapGetters, mapMutations } from 'vuex';
 import AddElement from 'tce-core/AddElement';
-import AssessmentItem from './AssessmentItem';
-import capitalize from 'lodash/capitalize';
+import AssessmentItem from '@/components/editor/structure/AssessmentItem';
 import cuid from 'cuid';
-import EventBus from 'EventBus';
+import filter from 'lodash/filter';
+import last from 'lodash/last';
 import map from 'lodash/map';
-
-const appChannel = EventBus.channel('app');
+import pickBy from 'lodash/pickBy';
 
 export default {
-  name: 'assessments',
+  name: 'assessment-block',
+  props: {
+    container: { type: Object, required: true },
+    tes: { type: Object, required: true }
```

Comment:
It can't be `contentElements` because in the `ContentContainers/index.vue` file the prop is still named [tes](https://github.com/ExtensionEngine/tailor/blob/develop/client/components/editor/structure/ContentContainers/index.vue#L26). I could change it but I think all of that should be done in a separate PR. If you want it to be a part of this one then I will change it.